docs/book/src/images/worker-machines-resources.plantuml

```diff
@@ -0,0 +1,48 @@
+@startuml worker-machines-resources
+
+skinparam nodesep 80
+skinparam ranksep 80
+skinparam object {
+    backgroundColor<< Infrastructure provider CRD >> #D9EAD3
+    backgroundColor<< Bootstrap provider CRD >> #FEF4F4
+}
+
+' Cluster API CRD
```

Comment:
Should we add the machine health check to this graph as well (given that it can monitor also worker machines) ?

Reply:
I added MachineHealthChecks at eaf710864.